test/hocr-eval-lines/hocr-eval-lines.tsht

```diff
@@ -4,9 +4,6 @@ TESTDATA="../testdata"
 plan 3
 
 exec_ok "hocr-eval-lines" "-v" "$TESTDATA/sample.txt" "$TESTDATA/sample.html"
-# Test code, remove it later ==>
-hocr-eval-lines "$TESTDATA/sample.txt" "$TESTDATA/tess.hocr"
-# <==
 
 # Currently Travis shows output like this:
 # ('segmentation_errors', 0)
```

Comment:
I guess also that this comment is not anymore true, because we switch to use `import_from_future` and therefore the print functions in Python 3 will also be printed without the brackets in Python 2.